DEVELOPER_GUIDE.md

```diff
@@ -22,6 +22,7 @@ So you want to contribute code to this project? Excellent! We're glad you're her
     - `cdk deploy OpenSearchMetricsNginxReadonly`: To deploy the dashboard read only setup.
     - `cdk deploy OpenSearchWAF`: To deploy the AWS WAF for the project ALB's.
     - `cdk deploy OpenSearchMetrics-Monitoring`: To deploy the alerting stack which will monitor the step functions and URL of the project coming from [METRICS_HOSTED_ZONE](https://github.com/opensearch-project/opensearch-metrics/blob/main/infrastructure/lib/enums/project.ts)
+    - `cdk deploy OpenSearchMetrics-GitHubApp`: Create the resources which launches the GitHub App. Listens to GitHub events and index the data to Metrics cluster.
```

Comment:
Change to OpenSearchMetrics-Automation-App?

Reply:
How about `OpenSearchMetrics-GitHub-Automation-App` so that folks know this is for running the GitHub automation app.

Reply:
Sure.